.github/workflows/main.yaml

```diff
@@ -17,6 +17,8 @@ jobs:
     - name: Setup yugabyte-db action
       id: server
       uses: ./
+      with:
+         yb_image_tag: 2.3.2.0-b37
```

Comment:
Should we use a more recent version of the YB image?

Reply:
The checks are failing because the `--advertise_address` does not exist in that particular version. 

```
Error: unrecognized arguments: --advertise_address 0.0.0.0.
                 {start,stop,destroy,status,version,collect_logs,connect,demo}
```